core/alpaca/alpaca_trade_manager.py

```diff
@@ -1,6 +1,8 @@
 from alpaca_trade_api.rest import REST, TimeFrame
 import pandas as pd
 from typing import Any, List, Optional
+from alpaca_trade_api.rest import APIError 
```

Comment:
You can append `APIError` to line 1 instead

Reply:
oh